initfiles/componentfiles/configschema/xsd/sparkthor.xsd

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema
+        xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
+        xmlns:hpcc="someuri">
+    <xs:include schemaLocation="types.xsd"/>
+    <hpcc:insert hpcc:schemaPath="/Environment/Software">
+        <xs:element name="SparkThorProcess" hpcc:itemType="sparkthor" hpcc:displayName="Spark Thor" maxOccurs="1" hpcc:class="component"
+                    hpcc:docid="Th.t1">
+            <xs:complexType>
+
+                <xs:sequence>
+                    <xs:element name="Instance" maxOccurs="unbounded" hpcc:class="elementSet"
```

Comment:
The new schema constructs might allow us to improve this portion of the spark-thor configuration...
I believe the intent here is to allow the admin to specify a cluster node as the spark master. By default, it should be tied to the ECLWatch ESP instance (if one exists)

Reply:
Let's change maxoccurs to 1, and use standard HW instance ref, remove dir attribute, not needed.

Reply:
Changes made.